docker/ccf.yaml

```diff
@@ -42,6 +42,8 @@ services:
     build:
       context: .
       dockerfile: Dockerfile.pdo-tp-ccf
+      args:
+        SGX_MODE: ${SGX_MODE:-SIM}
```

Comment:
is this always for the CHECK_ATTESTATION flag?

Reply:
thank you, yes you are right. CHECK_ATTESTATION is derived from SGX_MODE